pyface/ui/qt4/util/tests/test_gui_test_assistant.py

```diff
@@ -1,5 +1,6 @@
 import unittest
 
+from pyface.timer.api import CallbackTimer, do_after
```

Comment:
Unused import: I swapped out `do_after` for a use of the `CallbackTimer`, then forgot to remove `do_after`.